### ISSUE TITLE
Increasing training loss

### ISSUE THREAD

**Issue by PaulFonta** [PRBonn/PSPA#1]
Hello,

I am trying to train the PSPA model on a custom dataset. The only difference from Phenobench is that we have the weeds class as a semantic class and not instances.

The problem that I encounter is that the loss increases a lot after epoch 32.

Can I have an explanation of the cause of this, and could it be an issue with the configuration file?

![image](https://github.com/PRBonn/PSPA/assets/111424990/2ea65d97-09bd-48d0-b519-b6c6e4bb5226)

![Screenshot from 2024-07-04 09-43-04](https://github.com/PRBonn/PSPA/assets/111424990/64831332-45aa-4d2e-8f73-4fe6fcf11c58)
